onnxruntime/python/tools/transformers/benchmark_helper.py

```diff
@@ -164,9 +164,10 @@ def prepare_environment(cache_dir, output_dir, use_gpu, provider=None):
             ), "Please install onnxruntime-directml package to test GPU inference."
 
         else:
-            assert (
-                "CUDAExecutionProvider" in onnxruntime.get_available_providers()
-            ), "Please install onnxruntime-gpu package to test GPU inference."
+            assert any(
+                gpu_ep in onnxruntime.get_available_providers()
+                for gpu_ep in ["CUDAExecutionProvider", "ROCMExecutionProvider", "MIGraphXExecutionProvider"]
+            ), "Please install onnxruntime-gpu package, or install ROCm support, to test GPU inference."
```

Comment:
This have poor readibility. What you want is set disjoint test:
```py
set(onnxruntime.get_available_providers()).isdisjoint(["CUDAExecutionProvider", "ROCMExecutionProvider", "MIGraphXExecutionProvider"])
```

Reply:
```
>>> set(["CUDAExecutionProvider"]).isdisjoint(["CUDAExecutionProvider", "ROCMExecutionProvider", "MIGraphXExecutionProvider"])
False
>>> set(["CPUExecutionProvider"]).isdisjoint(["CUDAExecutionProvider", "ROCMExecutionProvider", "MIGraphXExecutionProvider"])
True
```

Reply:
Fair I appreciate that tip. I'll use this instead.

Reply:
Done